Client/TabManagement/Tab.swift

```diff
@@ -845,6 +841,19 @@ class Tab: NSObject {
 
         return .none
     }
+
+    // MARK: - ThemeApplicable
+
+    func applyTheme(theme: Theme) {
+        UITextField.appearance().keyboardAppearance = isPrivate ? .dark : (theme.type == .dark) ? .dark : .light
```

Comment:
To avoid checking for `(theme.type == .dark)`, we can add a new method on `ThemeType`, equivalent to `getBarStyle`. It could be `keyboardStyle(isPrivate: Bool)` or something like that 🤷‍♀️ so decision on how the keyboard is themed stays in the theme type.

Reply:
ok perfect